eng/DocsGenerator/DefinitionReference.template.md

```diff
@@ -74,6 +77,13 @@ Generated YAML:
 Currently, we don't support many marketplace tasks in C# as the project is still growing.
 If you find one useful, hit us up with a request, or better, with a pull request and we can add it to our library.
 
+Marketplace tasks can be supported and can be added to the organization if requested.
+
+Supported marketplace tasks:
+| Azure DevOps Extension | Repository | Nuget | Author |
+|------------------------|------------|---------|--------|
+| [JFrog Azure DevOps Extension](https://marketplace.visualstudio.com/items?itemName=JFrog.jfrog-azure-devops-extension) | [sharpliner/jfrog-extensions](https://github.com/sharpliner/jfrog-extensions) | | @flcdrg
```

Comment:
@flcdrg is the jfrog package published to nuget? https://github.com/sharpliner/jfrog-extensions/blob/64b382ff980a9413b6d17a6f9a480003296316f0/src/Tasks/Tasks.csproj#L7C19-L7C46

Reply:
@flcdrg currently the package is published from the profile of @premun , do we want to create a dedicated nuget profile named `Sharpliner` or something like this?

Reply:
I think that might be good to do, especially for things that come from the sharpliner GitHub org.

Could also look into reserving the Sharpliner prefix on NuGet too.

Reply:
Makes sense, I will set it up

Reply:
@flcdrg I pushed the package, that seemed to be the easiest: https://www.nuget.org/packages/Sharpliner.Extensions.JFrog/1.0.60

And I made you owner (I think you need to accept somewhere) so that you can use your own key.